src/Components/ContentSelection/ContentSelection.js

```diff
@@ -3,20 +3,30 @@ import style from './ContentSelection.module.scss'
 import { ViewContext } from '../../Context/ViewContext'
 import { VideoContext } from '../../Context/VideoContext'
 import { SettingsContext } from '../../Context/SettingsContext'
+import { SessionContext } from '../../Context/SessionContext'
 import { getRandomContent } from '../../apiCalls'
 import somaticContentIcon from '../../assets/content/somaticContentIcon.png'
 import meditationContentIcon from '../../assets/content/meditationContentIcon.png'
 import yogaContentIcon from '../../assets/content/yogaContentIcon.png'
 
 const ContentSelection = () => {
-  const [view, setView] = useContext(ViewContext)
-  const [videoLink, setVideoLink] = useContext(VideoContext)
+  const [ view, setView ] = useContext(ViewContext)
+  const [ videoLink, setVideoLink ] = useContext(VideoContext)
   const [ settings ] = useContext(SettingsContext)
+  const [ session, setSession ] = useContext(SessionContext)
+
+  const recordCategory = (category) => {
```

Comment:
I really like that we kept naming consistency with "record" in all the functions that are updating part of the session info - it makes it really easy for me to find these in all the various components

Reply:
Same, extra points for empathy on that decision in terms of being able to track down these related methods.